### PR TITLE
[NUXE-80] - Add 'close' option to kit, with examples

### PR DESCRIPTION
**Pull request by coleerikson**
#### Screens

click-to-close

#### Breaking Changes

No - New prop that is optional and defaults to existing state. 
_Please double check though : )_

#### Runway Ticket URL

[NUXE-80](https://nitro.powerhrg.com/runway/backlog_items/NUXE-80)

#### How to test this

Click toast examples in docs.

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `depreciation`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

**Comment by jasperfurniss** [comment]
There's some spacing differences between the react and rails examples, as well as a nice hover that only appears on the rails one. @coleerikson 
Screen Shot 2020-10-08 at 1 47 10 PM
Screen Shot 2020-10-08 at 1 47 03 PM

File: app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx [hunk 1]
```diff
@@ -14,36 +14,51 @@ const iconMap = {
...
+    closeable && 'remove_toast'
   )
   const icon = iconMap[status]
 
   return (
-    <div className={css}>
-      <If condition={icon}>
-        <Icon
-            className="pb_icon"
-            fixed_width
```
Comment (coleerikson) [inline app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx:37]: Fixed. This is a rails style prop. : /

Comment (jasperfurniss) [reply app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx:37]: Is this related to the failed milano build? Just checking. @coleerikson

Comment (coleerikson) [reply app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx:37]: I believe it's unrelated. Testing and rerunning Milano.

**Review by jasperfurniss — APPROVED** [review]

File: app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx [hunk 2]
```diff
@@ -14,36 +14,51 @@ const iconMap = {
...
 const FixedConfirmationToast = (props: FixedConfirmationToastProps) => {
-  const { className, status = 'neutral', text } = props
+  const [showToast, toggleToast] = useState(true)
+  const { className, closeable = false, status = 'neutral', text } = props
   const css = classnames(
     `pb_fixed_confirmation_toast_kit_${status}`,
     globalProps(props),
-    className
+    className,
+    { 'remove_toast': closeable }
```
Comment (jasperfurniss) [inline app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx:31]: yeah className needs to go last here. 

So i would do:
```
   { 'remove_toast': closeable },
    globalProps(props),
    className
```

Comment (jasperfurniss) [reply app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx:31]: and github was yelling at me about suggesting the commit-- hence i didn't do it.

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**